### ISSUE TITLE
Rename pros package to pros2

### ISSUE THREAD

**Issue by edjubuh** [purduesigbots/pros-atom#73]
Should be renamed to something else since the current name ("pros") makes it seem like this is the latest version.

**Comment by HotelCalifornia** [comment]
https://flight-manual.atom.io/behind-atom/sections/maintaining-your-packages/#rename-your-package

**Comment by HotelCalifornia** [comment]
may also want to rename/archive this repo as well, if we're not going to do any more work here (which is likely)

**Comment by Samuel12321** [comment]
#71 
#70

**Comment by HotelCalifornia** [comment]
@Samuel12321 yeah, potentially. Currently pros-bootstrapper@0.12 will install all the packages needed for the PROS 3 ecosystem, but it's not documented that well partially by design-- in general the only people who should be installing that package manually are those who know what they're doing (or following the instructions I linked to you in a previous comment)
